config/owasp/suppressions.xml

```diff
@@ -14,9 +14,14 @@
     <notes>Temporary Suppression
       CVE-2022-1471
       CVE-2023-28708
+      CVE-2023-20861
+      CVE-2023-20860
     </notes>
     <cve>CVE-2022-1471</cve>
     <cve>CVE-2023-28708</cve>
+    <cve>CVE-2023-20861</cve>
+    <cve>CVE-2023-20861</cve>
+    
```

Comment:
Put these in same order as they are listed in the notes section (which will be numerical order).

CVE-2023-20861 is listed twice - one should be CVE-2023-20860.

Remove blank line at end of section.